test_runner/fixtures/pageserver/utils.py

```diff
@@ -191,7 +191,11 @@ def wait_timeline_detail_404(
     tenant_id: TenantId,
     timeline_id: TimelineId,
     iterations: int,
+    interval: Optional[float] = None,
 ):
+    if interval is None:
```

Comment:
Cant we specify it as a default arg without option?

```python
interval: float = .25,
```

Reply:
The idea of using None is that it makes it easier for nested calls when we wrap it in some other function, and that other function can also have a `interval = None` arg, rather than having to know to omit the arg entirely.